src/pybind/mgr/volumes/fs/volume.py

```diff
@@ -62,11 +63,14 @@ def __init__(self, mgr):
         super().__init__(mgr)
         # volume specification
         self.volspec = VolSpec(mgr.rados.conf_get('client_snapdir'))
+
```

Comment:
unnecessary newline ?

Reply:
It's easier to read when related bits of code grouped in blocks and blocks are separated by a newline. This is a common practice, somehow we miss it Ceph for no reason.